conf/dovecot/dovecot.conf

```diff
@@ -57,7 +57,7 @@ userdb {
 
 userdb {
   driver = passwd-file
-  args = /etc/dovecot/app-senders-passwd
+  args = username_format=%n /etc/dovecot/app-senders-passwd
```

Comment:
Hmokay this is very obscure but apparently from the documentation ... 

>username_format=<s>: Look up usernames using this format instead of the full username (%u). If you want to enable [user@domain](mailto:user%40domain) logins but have only user in the file, set this to %n.

Are we sure this ain't going to create issues for app that are already configured to use the username and not user@domain ?

Reply:
Well, normally this should work. Tested with gitea with this config: https://github.com/Salamandar/gitea_ynh/blob/manifestv2/conf/app.ini#L36-L54

Reply:
Dovecot is an IMAP server. The config you're linking to is the SMTP config.

For the IMAP config of gitea, please see : https://docs.gitea.com/administration/config-cheat-sheet#incoming-email-emailincoming

Reply:
> Dovecot is an IMAP server. The config you're linking to is the SMTP config.
> 
> For the IMAP config of gitea, please see : https://docs.gitea.com/administration/config-cheat-sheet#incoming-email-emailincoming

Well I know that dovecot is IMAP server and I configured gitea to use dovecot as you linked the doc (and it worked greatly with this fix and the config that I make on gitea). But you justed droped anything that I did by force pushing on the PR. So yes now linked config is quite not valid...